core/src/actions/records.ts

```diff
@@ -206,6 +201,40 @@ export class RecordImport extends Action {
   }
 }
 
+export class RecordsImport extends Action {
+  permission: {
+    topic: ActionPermissionTopic;
+    mode: ActionPermissionMode;
+  };
+
+  constructor() {
+    super();
+    this.name = "records:import";
+    this.description = "fully import all records associated with a model";
+    this.outputExample = {};
+    this.permission = { topic: "record", mode: "write" };
+    this.middleware = ["authenticated-action"];
+    this.inputs = {
+      modelId: { required: true },
+    };
+  }
+
+  async run({ params }: { params: { modelId: string } }) {
+    let model: GrouparooModel;
+    let records: GrouparooRecord[];
+
+    await CLS.wrap(async () => {
+      model = await GrouparooModel.findById(params.modelId);
+      records = await GrouparooRecord.findAll({
+        where: { modelId: model.id },
```

Comment:
Do we even need to load the model if the modelId is in the params?

Reply:
Yeah - we wanted to validate that the modelId you provided exists.  Otherwise, if you sent `modelId=foo` you would get no error, and an empty list of Records back